.github/workflows/stale.yml

```
@@ -0,0 +1,39 @@
+name: 'Automatically close stale issues'
+
+on:
+    schedule:
+        # Runs every day at 8:00 AM CET
+        - cron: '0 7 * * *'
+    workflow_dispatch:
+
+permissions:
+    issues: write
+
+jobs:
+    stale:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/stale@v9
+              with:
+                  repo-token: ${{ secrets.GITHUB_TOKEN }}
+                  stale-issue-message: |
+                      This issue has been automatically marked as stale because it has not had recent activity and has the `awaiting-response` label.
```

nit: it's not (exactly) the same label as the one specified in `only-labels`
```suggestion
                      This issue has been automatically marked as stale because it has not had recent activity and has the `awaiting response` label.
```

---

Thanks!

Changes at [e825cf7](https://github.com/DataDog/dd-sdk-reactnative/pull/940/commits/e825cf75daa2b274753fa529cb74d48060a98810)